pkg/kubelet/prober/prober_manager.go

```diff
@@ -193,8 +192,8 @@ func (m *manager) AddPod(pod *v1.Pod) {
 		if c.LivenessProbe != nil {
 			key.probeType = liveness
 			if _, ok := m.workers[key]; ok {
-				klog.Errorf("Liveness probe already exists! %v - %v",
-					format.Pod(pod), c.Name)
+				klog.ErrorS(nil, "Liveness probe already exists for container",
```

Comment:
That 's will be result in
```
I0102 15:04:05.067890    1234 prober_manager.go:956] "Liveness probe already exists for container"
```
beacuse err == nil will result in [this](https://github.com/kubernetes/klog/blob/110d443dcd6b29c2941cabd9ab9d211063658a21/klog.go#L809)

Reply:
@yuzhiquan Problem you are mentioning should be fixed separately to migration. Let's continue with migration as described in migration guide.

Reply:
> @yuzhiquan Problem you are mentioning should be fixed separately to migration. Let's continue with migration as described in migration guide.

Sure.